integration/testdata/generate_pipeline/no_profiles/expectedSkaffold.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: skaffold/v1beta11
+kind: Config
+build:
+  artifacts:
+  - image: gcr.io/first-week-244218/getting-started
+deploy:
+  kubectl:
+    manifests:
+    - k8s-*
+profiles:
+- name: oncluster
+  build:
+    artifacts:
```

Comment:
@priyawadhwa is kaniko defined at per image level?
Is there a way to apply "onclluster" profile for all artifacts? Not for this phase, but for phase 2 when we support multiple artifacts, the config will become longer..

Reply:
yah, kaniko is defined per artifact. I'm not sure if there's any way to get around a longer config :/